api_end_end_testing/src/test/java/tds/user/api/tests/UserApiTests.java

```diff
@@ -0,0 +1,376 @@
+package tds.user.api.tests;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.jayway.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+
+import com.jayway.restassured.http.ContentType;
+import org.testng.annotations.Test;
+import tds.base.BaseUri;
+import tds.user.api.model.UserInfo;
+import tds.user.api.model.RoleAssociation;
+
+/*
+ * This class tests User API endpoints that creates, updates and deletes a user
+ */
+public class UserApiTests extends BaseUri {
+
+    private String uriLocation = "/rest/external/user";
```

Comment:
Minor, could we name this `userBaseUri`?   And do the same in the other test class that uses `uriLocation`